<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="FileView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>371</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="ArchievView">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>256</width>
      <height>321</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FormatType">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>7z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zip</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gzip</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zip2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tar</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="CompressType">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>440</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>mx0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mx1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mx5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mx7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mx9</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ArchivateButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Архивировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Посмотреть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>251</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> (по желанию)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
